kitsune/questions/templates/questions/new_question_react.html

```diff
@@ -6,6 +6,8 @@
 {% set title = _('Ask a Question') %}
 {% block breadcrumbs %}{% endblock %}
 
+{% set hide_signout_link = True %}
```

Comment:
you're right about the "Ask A Question" link on this page. we should probably also add:
`{% set hide_aaq_link = True %}`